query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_7434.rs

```diff
@@ -1,6 +1,10 @@
 use query_engine_tests::*;
 
-#[test_suite(schema(autoinc_id), capabilities(CreateMany, AutoIncrement), exclude(CockroachDb))]
+#[test_suite(
+    schema(autoinc_id),
+    capabilities(CreateMany, AutoIncrement),
+    exclude(CockroachDb, TiDB)
```

Comment:
Why does TiDB need to be excluded here?

Reply:
Oh sorry, I didn't set the `QUERY_BATCH_SIZE=10` environment variable in the goland run configuration, which caused the test to not throw an error as expected. I'll remove it.

Reply:
Fixed: https://github.com/prisma/prisma-engines/pull/3308/commits/4cf3f51c309dde001d4825cf6a54918572264090